src/main/resources/application.conf

```diff
@@ -1,41 +1,41 @@
 compendium {
+
   http {
     port = 8080
     host = "0.0.0.0"
   }
 
   protocols {
     storage {
-      # Choose storage type for protocol data
-      # Options: FILE, DATABASE
-      storage-type = "FILE"
-      # Path to folder where protocol contents will be stored
-      path = "/tmp/files"
-      path = ${?COMPENDIUM_PROTOCOLS_STORAGE_FILEPATH}
+      storage-type = "DATABASE"
+      storage-type = ${?COMPENDIUM_PROTOCOLS_STORAGE_TYPE}
+      jdbc-url = "jdbc:postgresql://localhost:5432/compendium_protocols"
+      jdbc-url = ${?COMPENDIUM_PROTOCOLS_STORAGE_JDBC_URL}
+      username = "postgres"
+      username = ${?COMPENDIUM_PROTOCOLS_STORAGE_USERNAME}
+      password = "postgres"
+      password = ${?COMPENDIUM_PROTOCOLS_STORAGE_PASSWORD}
+      driver = "org.postgresql.Driver"
+      connection-timeout = ${?COMPENDIUM_PROTOCOLS_STORAGE_CONNECTION_TIMEOUT}
+      idle-timeout = ${?COMPENDIUM_PROTOCOLS_STORAGE_IDLE_TIMEOUT}
+      max-lifetime = ${?COMPENDIUM_PROTOCOLS_STORAGE_MAX_LIFETIME}
+      minimum-idle = ${?COMPENDIUM_PROTOCOLS_STORAGE_MINIMUM_IDLE}
+      maximum-pool-size = ${?COMPENDIUM_PROTOCOLS_STORAGE_MAXIMUM_POOL_SIZE}
     }
 
-    # Change storage-type to DATABASE if you want to use this
+    # Change storage-type to FILE if you want to use this
     # and comment/delete storage properties above for FILE
-    #storage {
-    #  storage-type = "DATABASE"
-    #  jdbc-url = ""
-    #  jdbc-url = ${?COMPENDIUM_PROTOCOLS_STORAGE_JDBC_URL}
-    #  username = "postgres"
-    #  username = ${?COMPENDIUM_PROTOCOLS_STORAGE_USERNAME}
-    #  password = "postgres"
-    #  password = ${?COMPENDIUM_PROTOCOLS_STORAGE_PASSWORD}
-    #  driver = "org.postgresql.Driver"
-    #  connection-timeout = ${?COMPENDIUM_PROTOCOLS_STORAGE_CONNECTION_TIMEOUT}
-    #  idle-timeout = ${?COMPENDIUM_PROTOCOLS_STORAGE_IDLE_TIMEOUT}
-    #  max-lifetime = ${?COMPENDIUM_PROTOCOLS_STORAGE_MAX_LIFETIME}
-    #  minimum-idle = ${?COMPENDIUM_PROTOCOLS_STORAGE_MINIMUM_IDLE}
-    #  maximum-pool-size = ${?COMPENDIUM_PROTOCOLS_STORAGE_MAXIMUM_POOL_SIZE}
-    #}
+    # storage {
+    #   storage-type = "FILE"
+    #   storage-type = ${COMPENDIUM_PROTOCOLS_STORAGE_TYPE}
+    #   path = "/tmp/files"
+    #   path = ${?COMPENDIUM_PROTOCOLS_STORAGE_FILEPATH}
+    # }
```

Comment:
I would move the storage configuration comment into the .md, so we have a cleaner config file.

Reply:
As you may notice, I did not document `FILE` storage type for protocols, since I think it is not going to be the default option and we are not going to support it officially. 

IMHO, I don't think that having this comment in the `application.conf` is making it complicated right now. At some point, I'm sure it is not going to be there, but just for simplicity, let's keep it there in the meantime.